app/components/Card/CardFront.tsx

```diff
@@ -26,25 +27,25 @@ const CardFront: React.FC<CardFrontProps> = ({
   name,
   image,
   type,
+  path,
 }) => {
   // add a placeholder image and replace "default-image-url" with correct url
   const placeholderImage = "";
 
+  const style =
+    path === "/" ? responsiveValues.carousel : responsiveValues.machinesPage;
+
   const handleClick = () => {
     setIsFlipped(!isFlipped);
```

Comment:
update this to: `setIsFlipped((prevState) => !prevState);` to ensure that latest state is used (e.g. if user clicks rapidly)